src/js/editor/editor.js

```diff
@@ -640,8 +642,27 @@ class Editor {
     return false;
   }
 
+  handleCopy(event) {
+    const { clipboardData } = event;
+    const range = this.cursor.offsets;
+    const mobiledoc = this.post.cloneRange(range);
+    const html = `<div data-mobiledoc='${JSON.stringify(mobiledoc)}'></div>`;
+    clipboardData.setData('text/html', html);
+    event.preventDefault();
```

Comment:
Or, honestly, just letting natural browser copy do its thing for now would work prety well. We'll reparse the HTML on the way back in rather than using the already-serialized mobiledoc, but that doesn't seem to be a problem. Eventually we will want to keep that serialized mobiledoc so that cards can be preserved and copy/pasted around.